CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,24 @@
+# Code of Conduct
+
+The Twin Cities Aid Map community welcomes participation from everyone. 
```

Comment:
```suggestion
The Twin Cities Mutual Aid Map community welcomes participation from everyone. 
```

Should probably match the organization and the website url.

Reply:
Good catch! Thanks.